haiku-apps/desknotes/desknotes-1.1.recipe

```diff
@@ -1,6 +1,5 @@
 SUMMARY="A tool to put simple sticky notes on the Desktop"
-DESCRIPTION="
-With DeskNotes you can leave little sticky notes all over your desktop, \
+DESCRIPTION="With DeskNotes you can leave little sticky notes all over your desktop, \
 reminding you of things to do. The main difference between DeskNotes and \
 other note utilities for Haiku is, that DeskNotes is designed to be used \
```

Comment:
Should be:
```
DESCRIPTION="With DeskNotes you can leave little sticky notes all over your \
desktop, reminding you of things to do. The main difference between DeskNotes \
and other note utilities for Haiku is, that DeskNotes is designed to be used \
```
(Re-wrapping because the first line was > 80 chars. The rest is unchanged.)